test/toolkits/style_transfer/test_style_transfer.cxx

```diff
@@ -16,8 +16,85 @@
 
 #include "utils.hpp"
 
+/**
+ * Since the only backend is MPS, these tests won't pass on any system other
+ * than 10.15.
+ */
+#ifdef HAS_MACOS_10_15
+
 using namespace turi::style_transfer;
 
+
+BOOST_AUTO_TEST_CASE(test_init) {
```

Comment:
What does this test assert?

Reply:
I was presuming that no exceptions are thrown. I tried to add checks to see if the class was populated by checking the member variables but couldn't access them because they were private. Not exactly sure how to proceed.